### DISCUSSION TITLE
Builds still failing since incident 3/2/2023

### DISCUSSION THREAD

**Discussion by fulcrumlabsio** [community/community#49455] [Actions]
### Select Topic Area

Bug

### Body

I have a step in my Python/Django build with the following command:

```bash
sudo apt-get install libmemcached-dev --fix-missing
```

Since the incident on 3/2/2023, my builds have continually failed due to this 404 raised.

<img width="1246" alt="Screen Shot 2023-03-07 at 2 05 12 PM" src="https://user-images.githubusercontent.com/72598224/223564110-1f22dde6-7761-4da2-83cf-ee5a26da30f7.png">
<img width="1209" alt="Screen Shot 2023-03-07 at 2 05 21 PM" src="https://user-images.githubusercontent.com/72598224/223564124-f5aa6e87-9c6e-4632-bac1-00677f463de0.png">

I have not changed anything in my build steps. Any idea why I am still getting a 404? What's the workaround? Anyone else experiencing this?

**Comment by yesyouken** [comment]
I am also experiencing this with my builds

**Comment by vitalykarasik** [comment]
did you run 'apt update' before install?

**Comment by jge162** [comment]
Check and see if it is still in the repo @fulcrumlabsio.

`sudo apt-cache search libmemcached-dev`
